### PR TITLE
Feature/simple model

### PR DESCRIPTION
**Pull request by birdMarcus**
- implements a simple random forrest model
- adds the needed views for training

### CODE REVIEW

File: notebooks/nowcast.ipynb [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
```
Comment (birdNicolas) [inline notebooks/nowcast.ipynb:1]: do you really want to checkin the notebook? (would be the first notebook)

Comment (birdMarcus) [reply notebooks/nowcast.ipynb:1]: Muss nicht aus meiner Sicht, dann behalte ich das lokal.

File: infrastructure/database/templates/07_create_or_update_views.template.sql [hunk 2]
```diff
@@ -157,9 +182,17 @@ GROUP BY tree_id, "date";
...
 grant select on public.rainfall to web_anon;
+grant select on public.rainfall to qtrees_user;
 
 grant select on public.vector_tiles to authenticator;
 grant select on public.vector_tiles to web_anon;
+grant select on public.vector_tiles to qtrees_user;
+
+grant select on public.expert_dashboard to authenticator;
+grant select on public.expert_dashboard to web_anon;
+grant select on public.expert_dashboard to qtrees_user;
```
Comment (birdNicolas) [inline infrastructure/database/templates/07_create_or_update_views.template.sql:193]: should  `expert_dashboard` be exposed to `web_anon`

File: infrastructure/database/templates/08_sql_functions.template.sql [hunk 3]
```diff
@@ -50,5 +50,19 @@ BEGIN
 END
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION public.nowcast_input(_nowcast_date date, _type_id int)
```
Comment (birdNicolas) [inline infrastructure/database/templates/08_sql_functions.template.sql:53]: maybe call it `nowcast_inference_input`?

File: qtrees/modeling.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import geopandas as gpd
+import pandas as pd
+import io
+import requests
+import gzip
+import wradlib as wrl
+from shapely.geometry import Polygon
+import datetime
+from zipfile import ZipFile
+from requests.exceptions import RequestException
```
Comment (birdNicolas) [inline qtrees/modeling.py:10]: File rausnehmen?

File: qtrees/shading_index.py [hunk 5]
```diff
@@ -1,64 +1,70 @@
 from astral import LocationInfo
 from astral.sun import sun
 import datetime
-import xarray as xr
-from pyproj import Proj, transform
+import rioxarray
 import os
```
Comment (birdNicolas) [inline qtrees/shading_index.py:5]: remove file from PR

File: qtrees/validation_map.py [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+    # make it a gdf
+    vector_gdf = create_vector_gdf(df_shadow_index_vector)
+
+    # save to files
+    vector_gdf.to_file(vector_filepath, driver='ESRI Shapefile')
+
+    
+# create_raster_validation_seasons(target_map, gdf_original_trees, df_shadow_index, target_filepath, selected_seasons)
+create_vector_validation(df_shadow_original_index, gdf_original_trees, vector_original_filepath)
```
Comment (birdNicolas) [inline qtrees/validation_map.py:120]: gehört das hier rein?

File: infrastructure/database/templates/08_sql_functions.template.sql (hunk 3 above)
Comment (birdMarcus) [reply infrastructure/database/templates/08_sql_functions.template.sql]: renamed and added comment

File: qtrees/modeling.py (hunk 4 above)
Comment (birdMarcus) [reply qtrees/modeling.py]: Hast du raus genommen

**Pull request merged**